src/pick.c

```diff
@@ -460,7 +459,8 @@ selected_choice(void)
 				yscroll = selection = 0;
 			break;
 		default:
-			if (!isu8start(buf[0]) && !isprint(buf[0]))
+			if (length == 0
```

Comment:
I agree that zeroing out the input buffer isn't necessary. Thanks for catching that!

I'm not sure how the change to this condition is related to this. Could you elaborate on this? When is `length` zero?